tests/data/test_maestro.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+# encoding: utf-8
+#
+# Copyright 2022 Spotify AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+import numpy as np
+import os
+import pathlib
+from typing import List
+import wave
+
+import apache_beam as beam
+from apache_beam.testing.test_pipeline import TestPipeline
+
+from basic_pitch.data.datasets.maestro import (
+    MaestroToTfExample,
+    MaestroInvalidTracks,
+    create_input_data,
+)
+from basic_pitch.data.pipeline import WriteBatchToTfRecord
+
+RESOURCES_PATH = pathlib.Path(__file__).parent.parent / "resources"
+MAESTRO_TEST_DATA_PATH = RESOURCES_PATH / "data" / "maestro"
+
+TRAIN_TRACK_ID = "2004/MIDI-Unprocessed_SMF_05_R1_2004_01_ORIG_MID--AUDIO_05_R1_2004_03_Track03_wav"
```

Comment:
Is it possible to make a fake midif file and store it in a temporary directory for testing? Even if you had to make it with `python-midi` or equivalent?

Reply:
I'm not sure if the temp directory would work. I could try creating an empty fake midi file, but it's so small it kind of felt like a "why not"

Reply:
I can't find an easy way to reply to your comment above about the csv and json files, but mirdata was throwing an error if they weren't there.

Reply:
the wav files are being used for testing. I can try to mock them like I did for the 15m long test I guess

Reply:
I was able to mock all wav files + midi files and have the tests still pass. Removed data from the repo and refactored tests accordingly.